amanuensis/config-default.yaml

```diff
@@ -139,6 +139,10 @@ SUPPORT_EMAIL_FOR_ERRORS: null
 #   - Support `/data` endpoints
 # //////////////////////////////////////////////////////////////////////////////////////
 AWS_CREDENTIALS:
+  'AWS_SES':
+    aws_access_key_id: ""
+    aws_secret_access_key: ""
+    region_name: ""
```

Comment:
Unless you refactor the code everywhere AWS_SES is mentioned you shouldn't move it here. This may be a different ticket to organize and group all the AWS related configs under the same dictionary, but I would skip it here.